locales-da-DK.xml

```diff
@@ -54,18 +54,18 @@
       <multiple>refr.</multiple>
     </term>
     <term name="retrieved">hentet</term>
-    <term name="scale">scale</term>
+    <term name="scale">skala</term>
     <term name="version">version</term>
 
     <!-- ANNO DOMINI; BEFORE CHRIST -->
-    <term name="ad">e.Kr</term>
-    <term name="bc">f.Kr</term>
+    <term name="ad">e.v.t.</term>
+    <term name="bc">f.v.t.</term>
 
     <!-- PUNCTUATION -->
-    <term name="open-quote">«</term>
-    <term name="close-quote">»</term>
-    <term name="open-inner-quote">‘</term>
-    <term name="close-inner-quote">’</term>
+    <term name="open-quote">"</term>
+    <term name="close-quote">»"</term>
```

Comment:
Straight quotes are exceedingly rare, though. Are you sure? E.g. https://en.wikipedia.org/wiki/Quotation_mark shows guillemets (https://en.wikipedia.org/wiki/Guillemet) or curly quotes for Danish.

(cc @adam3smith)